docs/developers/v2/DEPLOYMENT.md

```diff
@@ -129,13 +129,15 @@ If you need a UI to test, you can coordinate with the strategists.
    ```
 
 1. Monitor `harvest` and `tend` triggers for first few days. Call `harvest`/`tend` manually.
+1. If this is a new vault deployment, test deposit, withdraw, and transfer to ensure functionality is as expected.
 
 ## Scaling up / Moving to Endorse
 
 In addition to the 2 strategists, a Core Developer has to review the strategy before going into production.
 
-1. Increase deposit limit according to the table [below](#Limits-per-Stage)
-1. Set management fee to production level:
+1. Consult with Yearn's web team to ensure that the new vault won't create any issues with the API and that all necessary token/vault metadata is ready. Additionally, if this vault needs to go in the "labs" category, then it needs to be manually added to the website.
```

Comment:
I think we need to find a way to avoid "consulting with anyone". Create an issue in the web repo on GH can be a good option